client/components/Contact.js

```diff
@@ -180,8 +179,7 @@ class Contact extends Component {
             <div className="c_contact__content__col">
               <img
                 className="c_contact__content__col__contact_image"
-                src={contactFormImageSrc}
-                srcSet={contactFormImageWebpSrc}
+                src={`${contactForm}?w=468`}
```

Comment:
Seems like this is duped from #161 but 🤷‍♂

Reply:
yup; this branch was made on top of #161